### ISSUE TITLE
[linefit] Replace linefit and improvedLinefit with the merged project (Trac #1603)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1595]
We need to make sure the default behavior is the same as previous linefit since we don't want to inadvertently change the behavior of any filters.  I think there was one parameter in particular, but I'm forgetting the details.

Migrated from https://code.icecube.wisc.edu/ticket/1603
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:55", 
    "description": "We need to make sure the default behavior is the same as previous linefit since we don't want to inadvertently change the behavior of any filters.  I think there was one parameter in particular, but I'm forgetting the details.", 
    "reporter": "olivas", 
    "cc": "gmaggi", 
    "resolution": "fixed", 
    "_ts": "1550067295757382", 
    "component": "combo reconstruction", 
    "summary": "[linefit] Replace linefit and improvedLinefit with the merged project", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-03-24T14:11:10", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160328T17:45:34: gmaggi commented:

> I think the parameter is this one: http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/sandbox/gmaggi/icerec/linefit2_v3/private/linefit/I3LineFit.cxx#L89
> 
> That runs the LineFit only, without the cleaning made by improvedLineFit.
>

**Comment by icecube-trac** [comment]
Trac update at 20160412T21:59:26: 
* olivas changed milestone from "IceRec V5" to "IceRec V5.1"
* olivas commented:

> Going to have to put this off until next season.  I count 105 references to improvedLinefit, which was considerably more extensive than I thought.  So the new plan is: 
> 1) Deprecate linefit and improvedLinefit after the IceRec V5 release.
> 2) Rename this linefit so the names don't clash and they can all live side-by-side in combo.
> 3) Over the next year adapt each project that uses improvedLinefit and linefit to use the new project.

**Comment by icecube-trac** [comment]
Trac update at 20160722T22:02:06: 
* olivas changed owner from "olivas" to "kjmeagher"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20160726T15:34:49: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "assigned" to "closed"
* kjmeagher commented:

> In changeset 148760:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="148760"
> After testing I certify that these changes do not effect the performance of LineFit, merging gmaggi's changes into the trunk. Fixes #1595
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:55: 
* olivas commented:

> Milestone IceRec 5.2 deleted


* olivas changed milestone from "IceRec 5.2" to ""
